example/w-flash/README.md

```diff
@@ -48,6 +48,8 @@ into cookies and reads them back out:
 
 ```ocaml
 let () =
+  Dream.initialize_log ~level:`Info ();
```

Comment:
AFAIK this call shouldn't be necessary. Is it? If `set_log_level` requires it, `set_log_level` should be altered to do a lazy internal `initialize_log`.

Reply:
When I tested this against the `w-flash` example it did appear to be necessary; unfortunately I don't fully understand why. I've updated `set_log_level`.